<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Executor de Formulas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>92</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalc">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>92</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEnum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Defina as variáveis </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVar1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Variável 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVar2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Variável 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVar3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Variável 3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leVar2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>113</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leVar3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>113</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leVar1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>113</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fórmulas:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>109</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selecione sua opção&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>109</width>
      <height>26</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Unidade de Medida</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDebug">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>92</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>109</width>
      <height>26</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConvert">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>92</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Converter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSobre"/>
    <addaction name="separator"/>
    <addaction name="aMenuDebug"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="aMenuDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug mode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
